components/bases-locales/validator/report/row-issues.js

```diff
@@ -7,12 +7,12 @@ const RowErrors = ({errors, warnings, field}) => (
   <div className='abnormalities'>
     <h3>Anomalie{(errors.length + warnings.length) > 1 ? 's' : ''} :</h3>
     <div className='error-list'>
-      {warnings.map(err => (
-        <div key={err} className={`issue warning ${field && (field.warnings.includes(err)) ? 'select' : ''}`}>
+      {warnings.map((err, i) => (
```

Comment:
Lint: Do not use Array index in keys

Reply:
On peut simplement muter le linter dans ce genre de cas.

Reply:
Par la suite on aura des warnings et errors plus riches, mais là on va zapper...